### ISSUE TITLE
Support suite and target config from config file

### ISSUE THREAD

**Issue by phuu** [phuu/integrator#41]
```js
{
  default: {
    suite: 'suite.js',
    configuration: 'browserstack'
  }
}
```

**Comment by phuu** [comment]
Initially just going to do this for suite — configuration is likely to change much more (so it should be under a `default` key) but suite isn't:

```js
{
  suite: 'suite.js',
  default: {
    configuration: 'browserstack'
  },
  configurations: {
    browserstack: { ... }
  }
}
```

**Comment by phuu** [comment]
Will also need `default.action`?

**Comment by phuu** [comment]
Addressed by 5466fe5.
